docs/src/examples/components/Toolbar/Usage/ToolbarExampleWithTooltip.shorthand.tsx

```diff
@@ -44,10 +44,14 @@ const ToolbarExampleShorthand = () => {
           tooltip: 'More options',
           menu: [
             {
+              wrapper: undefined, // to avoid issue with keys
```

Comment:
let's try to find a better solution - how would a developer know that he should use wrapper: undefined?

Reply:
I can no longer reproduce any errors without setting `wrapper` to `undefined`. Removed.